lighthouse-core/gather/gatherers/viewport-dimensions.js

```diff
@@ -21,18 +21,21 @@ const Gatherer = require('./gatherer');
 /* global window */
 
 /* istanbul ignore next */
-function getContentWidth() {
+function getViewportDimensions() {
   // window.innerWidth to get the scrollable size of the window (irrespective of zoom)
   // window.outerWidth to get the size of the visible area
   // window.devicePixelRatio to get ratio of logical pixels to physical pixels
   return Promise.resolve({
+    scrollPosition: {x: window.scrollX, y: window.scrollY},
     scrollWidth: window.innerWidth,
+    scrollHeight: window.innerHeight,
```

Comment:
Ah I see these now. Yea, I would switch these to use the same name as the property you're stashing.

I typically use `document.documentElement.clientWidth` to get the viewport width, but inner* works great. That's what you want.